src/mesh_sandbox/routers/admin.py

```diff
@@ -135,3 +140,32 @@ async def get_mailbox_details(
 ) -> MailboxDetails:
     mailbox = await handler.get_mailbox_details(mailbox_id)
     return mailbox
+
+
+@router.get(
+    "/admin/message/{message_id}",
+    summary=f"Get message details matching id from message store. {TESTING_ONLY}",
+    status_code=status.HTTP_200_OK,
+    response_model=MessageDetails,
+    response_model_exclude_none=True,
+)
+@router.get(
+    "/messageexchange/admin/message/{message_id}",
+    summary=f"Get message details matching id from message store. {TESTING_ONLY}",
+    status_code=status.HTTP_200_OK,
+    response_model=MessageDetails,
+    response_model_exclude_none=True,
+)
+@router.get(
+    "/messageexchange/message/{message_id}",
```

Comment:
i don't think this one should exist. it's fine for get_mailbox_details because that actually is part of the mesh api.